src/HealthGPS/dummy_model.cpp

```diff
@@ -0,0 +1,75 @@
+#include "dummy_model.h"
```

Comment:
What convention are we following on adding docstrings in header files or code? I've seen [this comment](https://stackoverflow.com/a/3572839/3778792) making a distinction. I don't mind as long as things are commented.

Reply:
The convention I have been following here is to include docstrings in headers only, and to have inline comments in the CPP file code. Generally a matter of taste, but having them in both means needing to ensure that both are up to date, as opposed to just needing to keep header docstrings up to date. I'm open to being convinced otherwise, but that has struck me as a good compromise so far.

I see a few fn declarations in headers that have been missed. I'll make sure they're added.